content/attacks/posts/2011-10-05-Bitcoin7.md

```diff
@@ -0,0 +1,33 @@
+---
+date: 2011-10-05
+target-entities: Bitcoin7
+entity-types:
+  - Exchange
+  - Custodian
+attack-types:
+  - Wallet Hack
+  - Web Infrastructure Attack
+title: Bitcoin7 Hacked, Losing 5000 BTC
+---
+
+## Summary
+
+On October 5, 2011, Bitcoin7, a cryptocurrency exchange, suffered a security breach that resulted in the loss of approximately 5,000 BTC. The attack was facilitated by a combination of weak security practices and system vulnerabilities](https://bitcointalk.org/index.php?topic=46982.0), which allowed the attackers to gain unauthorized access to the exchange's hot wallet. Bitcoin7 claimed to have compensated affected customers and improved security measures following the attack, resuming normal operations within a few days.
```

Comment:
I assume `pcombination` should be replaced with `[combination` since the link lacks opening bracket

Reply:
💯